src/components/Table/table.scss

```diff
@@ -1,4 +1,5 @@
 .react-table-wrapper {
+  height: 100%;
```

Comment:
not sure about this one also - since we adjust table height in the components that use Table. correct me if I'm wrong

Reply:
updated